.github/workflows/docker_build.yml

```diff
@@ -84,12 +89,44 @@ jobs:
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD}}
 
-      - name: Build and push
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ inputs.docker_context }}
+          file: ${{ inputs.docker_file }}
+          builder: ${{ steps.buildx.outputs.name }}
+          load: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+            OMNI_COMPONENT=${{ env.COMPONENT_NAME }}
+            NODE_ENV=${{ inputs.node_env }}
+            NPM_TOKEN=${{ secrets.NPM_TOKEN }}
+            OMNI_COMPONENT_COMMIT=${{ github.sha }}
+            OMNI_COMPONENT_VERSION=${{ steps.meta.outputs.version }}
+      
+      - name: Set docker image tar version
+        id: setimgver
+        run: |
+          calculatedSha=$(git rev-parse --short ${{ github.sha }})
+          echo "::set-output name=IMAGE_TAR_VER::$calculatedSha"
+          
+      - name: save docker image to tar file
+        run: docker save --output ${{ inputs.project }}-${{ env.COMPONENT_NAME }}-${{ steps.setimgver.outputs.IMAGE_TAR_VER }}.tar ${{ steps.meta.outputs.tags }}
```

Comment:
use `outputs` https://github.com/marketplace/actions/build-and-push-docker-images#inputs  to get the tar file instead of `docker save`